src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/SearchSortEnum.cs

```diff
@@ -0,0 +1,16 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.OperationalInsights.Models
+{
+
+    /// <summary>
+    /// Defines values for SearchSortEnum.
+    /// </summary>
+    public static class SearchSortEnum
+    {
+        public const string Asc = "asc";
```

Comment:
This matches the current implementation (which uses asc and desc in the payload). Do you mean to change it like this:
public const string Ascending = "asc"; 

If so, what swagger extension I can use to change the constant name? This is what the current definition looks like:
"enum": [
            "asc",
            "desc"
          ],
          "x-ms-enum": {
            "name": "SearchSortEnum",
            "modelAsString": true
          }